packages/modules/src/BatchedSessionRouterModule.ts

```diff
@@ -62,7 +62,6 @@ export class BatchedSessionRouterModule extends BaseValidationModule {
       const sessionModule = await SessionKeyManagerModule.create({
         moduleAddress: instance.sessionManagerModuleAddress,
         smartAccountAddress: moduleConfig.smartAccountAddress,
-        nodeClientUrl: moduleConfig.nodeClientUrl,
```

Comment:
this was optional anyway right?

Reply:
Yeah I did ask Aman about this, he mentioned that it could be removed. It didn't look to be used anywhere